charts/scorpio-broker-dist/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if not (lookup "v1" "Namespace" .Values.namespace .Values.namespace ) }}
```

Comment:
Why is this required? Namespaces are usually handled by helm.

Reply:
We have incorporated the review comment. Please review again

Reply:
Removed namespace from the files, Please review again